src/assets/sass/components/_navbar.scss

```diff
@@ -16,6 +16,11 @@ nav {
   & > .nav-right {
     float: right;
     padding: 0.7rem 1.5rem 0;
+
+    .active {
```

Comment:
There might be a more generic place for this, but many of the other places I tried led to the style being overridden due to CSS priority rules and an `!important` didn't seem right.